io.github._0xzer0x.qurancompanion.yml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github._0xzer0x.qurancompanion
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.kde.Sdk.Locale
+command: quran-companion
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.gtk.vfs.*
+modules:
+  - name: platform-bootstrap
+    buildsystem: simple
+    build-commands:
+      - install -D logo64.png /app/share/icons/hicolor/64x64/apps/io.github._0xzer0x.qurancompanion.png
+      - install -D logo128.png /app/share/icons/hicolor/128x128/apps/io.github._0xzer0x.qurancompanion.png
+      - install -D logo256.png /app/share/icons/hicolor/256x256/apps/io.github._0xzer0x.qurancompanion.png
+      - install -D io.github._0xzer0x.qurancompanion.metainfo.xml /app/share/metainfo/io.github._0xzer0x.qurancompanion.metainfo.xml
+      - install -D io.github._0xzer0x.qurancompanion.desktop /app/share/applications/io.github._0xzer0x.qurancompanion.desktop
+    sources:
+      - type: file
+        path: logo64.png
+      - type: file
+        path: logo128.png
+      - type: file
+        path: logo256.png
```

Comment:
since these don't come from upstream (they should) only the biggest size is necessary.

Reply:
done.